### ISSUE TITLE
After renaming the variable with the new version, the next open does not take effect.[feature]

### ISSUE THREAD

**Issue by fengcoem** [skylot/jadx#1474]
*Describe your idea:*
First of all, I would like to thank the author for providing a powerful tool for everyone to use. I encountered some problems when updating to the latest version. After renaming the variable field, method or class name, it is only valid for this time. When I open it next time,  all of  the last time renamed variable do not work. I don't know if there is a problem with the settings or if it is a bug. When I use version 1.2.0 it is fine. I hope the author can fix it ,thanks a lot.

**Comment by skylot** [comment]
@fengcoem  do you save project file after renames?
Now (after dfdc14ea8618a7242d04bd6ea066d3e6b60a10b9 by #1022) jadx stores all user renames in project file, so you need to save project and next open it instead.
If this is not your case, please share more info about your workflow :slightly_smiling_face:

**Comment by fengcoem** [comment]
I usually use the jadx-gui.bat script to open an apk file, rename the variable and save it. The renamed variables are found in the .jobf file are also correct. When I opened this apk next time, I found my local  .jobf file were overwritten instead of reload. Before, the old version used this workflow and everything was fine. By the way, I don't change the options panel. I would like to know how to do it with work correctly just like older version.
